conda/install.py

```diff
@@ -598,6 +605,14 @@ def unlink(prefix, dist):
         with open(meta_path) as fi:
             meta = json.load(fi)
 
+        # TODO Refactor to if not pkg.is_writable
+        if on_win and not can_open_all_files_in_prefix(prefix, meta['files']):
+            # TODO Should eventually return rather than exiting
+            sys.exit(
```

Comment:
Are all unlink instructions batched together or do we unlink then link each package?